CONTRIBUTING.md

```diff
@@ -34,90 +34,107 @@ These are one time installations required to be able to test your changes locall
 1. Install [Node.js](https://nodejs.org/en/download/) for your platform
 1. Install [VS Code](https://code.visualstudio.com/download) for your platform
 1. Install the dependencies. From the repository root run:
+
 ```bash
-$ npm i
+npm i
 ```
 
-## Making & testing changes
+## Dev loop & Testing changes
+
+The extension is written in TypeScript and built using [webpack](https://webpack.js.org/). The dev loop steps and commands have been tested on both Mac and Windows.
+
+1. Go to the Debug tab.
+    1. Hit `Watch all & Launch Extension (workspace)` if you want to work on the main VS Code extension like the left sidebar and the UI for the extension.
+    1. Hit `Watch & Launch Extension + language-server (workspace)` If you want to work on the language services code and want to debug and work on the hover, syntax highlighting, and other functionality within the Workflow files.
+        * This will attach to an instance of the language server running on port `6010`
+1. Hit the green button (this will automatically run `npm watch` for you and monitor for changes) which will open a local version of the extension using the _extension development host_.
```

Comment:
```suggestion
1. Hit the play button (this will automatically run `npm watch` for you and monitor for changes) which will open a local version of the extension using the _extension development host_.
```